src/cli/record/delete.ts

```diff
@@ -134,6 +155,24 @@ const handler = async (args: Args) => {
   return undefined;
 };
 
+const hasApiToken = (apiTokenArg?: string | string[]): boolean => {
+  if (!apiTokenArg) {
+    return false;
+  }
+
+  if (typeof apiTokenArg === "string") {
+    return !!apiTokenArg;
+  }
+
+  if (apiTokenArg.length === 0) {
+    return false;
+  }
+
+  const apiToken = apiTokenArg.filter(Boolean);
+
+  return apiToken && apiToken.length > 0;
```

Comment:
You can simplify this section.

```suggestion
  return apiTokenArg.filter(Boolean).length > 0;
```